### PR TITLE
AWS Lambda, serverless Framework package pattern: Update for prisma 3

### PR DESCRIPTION
**Pull request by jsw**
## Describe this PR

Update the serverless package patterns for prisma 3. 

See https://github.com/prisma/prisma/issues/6644#issuecomment-953359344

## Changes

Update serverless package patterns

## What issue does this fix?

Reduces deployment size

### CODE REVIEW

**Comment by jsw** [comment]
cc @janpio

File: content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx [hunk 1]
```diff
@@ -334,8 +334,11 @@ The Serverless configuration file includes a package pattern that excludes all P
 ```
 package:
   patterns:
+    - '!node_modules/@prisma/engines/**'
     - '!node_modules/.prisma/client/libquery_engine-*'
     - 'node_modules/.prisma/client/libquery_engine-rhel-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - 'node_modules/prisma/libquery_engine-rhel-*'
```
Comment (janpio) [inline content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx]:
```suggestion
    - '!node_modules/.prisma/client/libquery_engine-*'
    - 'node_modules/.prisma/client/libquery_engine-rhel-*'
    - '!node_modules/prisma/libquery_engine-*'
    - 'node_modules/prisma/libquery_engine-rhel-*'
    - '!node_modules/@prisma/engines/**'
```
Let's change the order a bit: I moved the engines pattern to the end instead the first line.

That makes:
- First two lines about Prisma Client that should indeed contains no query engines besides the Red Hat one.
- Last line about all the Engines in @prisma/engines that are not required.

Why do you want to include the Query Engine of the CLI though?

Comment (jsw) [reply content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx]: @janpio You're saying that `node_modules/prisma/libquery_engine-rhel-*` is not needed? I didn't know it is for the CLI, I was just blindly including everything Red Hat. If so, I will incorporate your reordering suggestion and simplify further.

Comment (janpio) [reply content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx]: I don't think it is, unless the `prisma` CLI is invoked and included. `node_modules/prisma` itself is only present when the `prisma` package is installed, and probably the Engine file is copied over there at some point in the process.

**Comment by jsw** [comment]
I updated my build to what's in the (updated) PR and it seems to work 👍

**Review by janpio — APPROVED** [review]

**Comment by tanberry** [comment]
Thank you @jsw !! Your contributions to the code and the docs are much appreciated.

**Pull request merged**